src/routes/news/NewsPage.tsx

```diff
@@ -0,0 +1,106 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import styled from "styled-components";
+import LoadingProgress from "../../shared/components/LoadingProgress";
+import Select from "../../shared/components/Select";
+import { H2, screenMdMin } from "../../shared/styles";
+import NewsCard, { NewsArticle } from "./components/NewsCard";
+
+const NEWS_SELECT_OPTIONS = [
+  {
+    name: "Nutrition",
+  },
+  {
+    name: "Pharmaceuticals",
+  },
+  {
+    name: "Health",
+  },
+];
```

Comment:
Can this just be an array of string or are you planning on adding other properties to it later?